code/modules/antagonists/heretic/items/eldritch_painting.dm

```diff
@@ -0,0 +1,418 @@
+
+// The sister and He Who Wept eldritch painting
+// All eldritch paintings are based on this one, with some light changes
+/obj/item/wallframe/painting/eldritch
+	name = "The sister and He Who Wept"
+	desc = "A beautiful artwork depicting a fair lady and HIM, HE WEEPS, I WILL SEE HIM AGAIN."
+	icon = 'icons/obj/signs.dmi'
+	resistance_flags = FLAMMABLE
+	flags_1 = NONE
+	icon_state = "eldritch_painting_weeping"
+	result_path = /obj/structure/sign/painting/eldritch
+	pixel_shift = 30
+
+/obj/structure/sign/painting/eldritch
+	name = "The sister and He Who Wept"
+	desc = "A beautiful artwork depicting a fair lady and HIM, HE WEEPS, I WILL SEE HIM AGAIN. Destroyable with wirecutters."
+	icon = 'icons/obj/signs.dmi'
+	icon_state = "eldritch_painting_weeping"
+	custom_materials = list(/datum/material/wood =SHEET_MATERIAL_AMOUNT)
+	resistance_flags = FLAMMABLE
+	buildable_sign = FALSE
+	// The list of canvas types accepted by this frame, set to zero here
+	accepted_canvas_types = list()
+	// This stops people hiding their sneaky posters behind signs
+	layer = CORGI_ASS_PIN_LAYER
```

Comment:
This layer... lol

Reply:
left behind from me doing something silly